query/simplequery/query_test.go

```diff
@@ -704,11 +704,7 @@ func TestCornerCases(t *testing.T) {
 		q.requestError(ctx, node1.ID(), errors.New(""))
 	}))
 
-	// create simulator
-	s := sim.NewLiteSimulator(clk)
-	sim.AddSchedulers(s, sched0)
-	// run simulation
-	s.Run(ctx)
+	sched0.RunOne(ctx)
```

Comment:
This isn't doing the same as before. It was running the full simulation, and it is now running a single step.

Reply:
Yep. Still passed the test but that's irrelevant since this PR isn't needed